data-es2016plus.js

```diff
@@ -37,6 +37,7 @@ exports.tests = [
           safaritp: true,
           webkit: true,
           duktape20: true,
+          duktape21: true,
```

Comment:
You don't need to add every version here.  What is true for `duktape20` is also true for `duktape20` and all following versions.